src/Blocks/components/table/table-admin.scss

```
@@ -0,0 +1,28 @@
+.es-table {
+	position: relative;
+	overflow: auto;
+
+	table {
+		width: 100%;
+		border-spacing: 0;
+	}
+
+	thead {
+		background-color: var(--global-colors-esf-gray-100);
+	}
+
+	th {
+		column-span: none;
+		padding: 0.5rem;
+		text-align: left;
+		border-right: 1px solid var(--global-colors-esf-gray-200);
+		border-bottom: 1px solid var(--global-colors-esf-gray-200);
```

```suggestion
		border-inline-end: 1px solid var(--global-colors-esf-gray-200);
		border-block-end: 1px solid var(--global-colors-esf-gray-200);
```

Maybe use CSS logical properties to make code more reusable?

---

the CSS will be chaned full to tailwind in the later iterations so this is a temp setup until this is done. tnx for the suggestions